.travis.yml

```diff
@@ -4,4 +4,4 @@ addons:
 node_js:
   - "6"
   - "8"
-script: yarn lint && yarn test
+script: yarn lint && yarn test && yarn add --dev webpack@3 && yarn test
```

Comment:
You can do this in the matrix instead and run them in parallel, rather than in serial